### PR TITLE
[21] La pantalla principal crashea la app cuando el usuario scrolea.

### PR DESCRIPTION
**Pull request by jinmyjbv**
# Description

Code added to fix the  bug in the main screen caused by blah blah blah, that's why I added whatever I added

# Change Type

- Bug fix, no breaking change.

# Steps to test

Open app -> navigate to main screen -> activate don't keep activities in dev tools -> scroll

# Prove

<screenshot 1>

### CODE REVIEW

**Comment by jinmyjbv** [comment]
I don't think that `code cleaning` is a descriptive enough name for the commit you made there .

**Comment by jinmyjbv** [comment]
I think you should add an additional screenshot for whatever. I don't like the one you added.

File: app/build.gradle [hunk 1]
```diff
@@ -54,18 +54,6 @@ dependencies {
...
-    implementation "androidx.lifecycle:lifecycle-livedata-ktx:$lifecycle_version"
-    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:$lifecycle_version"
-    implementation 'androidx.navigation:navigation-fragment-ktx:2.2.1'
-    implementation 'androidx.navigation:navigation-ui-ktx:2.2.1'
-    implementation 'com.squareup.retrofit2:retrofit:2.6.2'
-    implementation 'com.squareup.retrofit2:converter-gson:2.6.2'
-    implementation 'com.github.PhilJay:MPAndroidChart:v3.0.3'
-    implementation 'com.squareup.picasso:picasso:2.71828'
-    implementation "androidx.room:room-runtime:$room_version"
-    kapt "androidx.room:room-compiler:$room_version"
```
Comment (jinmyjbv) [inline app/build.gradle]: why?

File: app/src/main/java/com/example/ncov19traking/daos/CountryDao.kt [hunk 2]
```diff
@@ -11,9 +11,7 @@ interface CountryDao {
     @Insert(onConflict = REPLACE)
     fun save(numbersByCountry : Array<NumbersByCountry>)
 
-    @Query("SELECT * FROM NumbersByCountry")
+    @Query("SELECT * FROM NumbersByCountry WHEN yes")
```
Comment (jinmyjbv) [inline app/src/main/java/com/example/ncov19traking/daos/CountryDao.kt]: This won't compile. Why do you think we need that yes WHEN check?

File: app/src/main/java/com/example/ncov19traking/daos/CountryDao.kt [hunk 3]
```diff
@@ -11,9 +11,7 @@ interface CountryDao {
...
     fun load(): Array<NumbersByCountry>
 
-    @Query("DELETE FROM NumbersByCountry")
-    fun delete()
```
Comment (jinmyjbv) [inline app/src/main/java/com/example/ncov19traking/daos/CountryDao.kt]: We are using this delete Method in the presenter. Add it back.

File: app/src/main/java/com/example/ncov19traking/data/NCoVDataBase.kt [hunk 4]
```diff
@@ -14,7 +14,7 @@ import com.example.ncov19traking.models.NumbersByCountry
 import com.example.ncov19traking.models.Timeline
 
 @Database(
-    entities = [NCoVInfo::class, NCoVInfoYesterday::class, NumbersByCountry::class, Timeline::class],
```
Comment (jinmyjbv) [inline app/src/main/java/com/example/ncov19traking/data/NCoVDataBase.kt]: ?

File: app/build.gradle (hunk 1 above)
Comment (jinmyjbv) [reply app/build.gradle]: done

File: app/src/main/java/com/example/ncov19traking/daos/CountryDao.kt (hunk 2 above)
Comment (jinmyjbv) [reply app/src/main/java/com/example/ncov19traking/daos/CountryDao.kt]: done

File: app/src/main/java/com/example/ncov19traking/daos/CountryDao.kt (hunk 3 above)
Comment (jinmyjbv) [reply app/src/main/java/com/example/ncov19traking/daos/CountryDao.kt]: done

File: app/src/main/java/com/example/ncov19traking/data/NCoVDataBase.kt (hunk 4 above)
Comment (jinmyjbv) [reply app/src/main/java/com/example/ncov19traking/data/NCoVDataBase.kt]: done

File: app/src/main/java/com/example/ncov19traking/data/NCoVDao.kt [hunk 5]
```diff
@@ -1,26 +0,0 @@
-package com.example.ncov19traking.data
```
Comment (ruif3r) [inline app/src/main/java/com/example/ncov19traking/data/NCoVDao.kt]: why you delete this class?

**Review by ruif3r — CHANGES_REQUESTED** [review]

**Review by ruif3r — APPROVED** [review]
ok

**Pull request merged**